package.json

```diff
@@ -217,6 +217,7 @@
     "uid-number": "0.0.6",
     "uniqid": "5.3.0",
     "uuid": "8.3.2",
+    "v8-compile-cache": "2.3.0",
```

Comment:
This is used in code but wasn't added as a dependency, so with the non-hoisted node_modules layout it caused issues.